.github/workflows/turing.yaml

```diff
@@ -27,7 +27,6 @@ on:
       - "docs/**"
       - "engines/pyfunc-ensembler-job/**"
       - "engines/pyfunc-ensembler-service/**"
-      - "sdk/**"
```

Comment:
If only the SDK files are modified, why should we run the whole turing workflow? Is it for the e2e tests?

Reply:
Yep, it's to trigger the e2e tests (we want to check if the existing workflows still work after changes to the SDK are made) but come to think of it, we can perhaps create a new `turing/e2e/**` directory and dump both the API e2e and SDK tests there, if that helps managing the CI/CD workflows more easily?